src/app.rs

```diff
@@ -441,6 +442,8 @@ impl<S, E: Send + Sync + 'static> ApplicationBuilder<S, E> {
         world.add_resource(Stopwatch::default());
         world.add_resource(Time::default());
 
+        world.register::<Named>();
```

Comment:
Why do we need this? This is only needed if you never intend to use the component in a system, and only from the states. If we're serious about doing this for Named, we should do it for a whole bunch of other components tbh.

Reply:
I explained most of my thinking in the initial comment. Basically, it's because a user may name an entity without actually using the named component in a system which means the user has to register the named component by themselves until they do decide to add a system which does use it. What other components are you thinking should also be handled this way?

Reply:
I'm fine with it being added.